<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_session"/>
   <addaction name="actionStays_on_top"/>
   <addaction name="actionOptions"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionNew_session"/>
    <addaction name="actionClose_all_sessions"/>
    <addaction name="actionSave_all_sessions"/>
    <addaction name="actionLoad_sessions"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actiontoolBar"/>
    <addaction name="actionStays_on_top"/>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuViews"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionStays_on_top">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/unpin.png</normaloff>
     <normalon>:/images/pin.png</normalon>:/images/unpin.png</iconset>
   </property>
   <property name="text">
    <string>Stays on top</string>
   </property>
  </action>
  <action name="actionNew_session">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/add_new.png</normaloff>:/images/add_new.png</iconset>
   </property>
   <property name="text">
    <string>New session</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actiontoolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>toolBar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose_all_sessions">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/close_all.png</normaloff>:/images/close_all.png</iconset>
   </property>
   <property name="text">
    <string>Close all sessions</string>
   </property>
  </action>
  <action name="actionSave_all_sessions">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/save_all.png</normaloff>:/images/save_all.png</iconset>
   </property>
   <property name="text">
    <string>Save all sessions</string>
   </property>
  </action>
  <action name="actionLoad_sessions">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Load sessions</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources/serial-tool.qrc">
     <normaloff>:/images/close_window.png</normaloff>:/images/close_window.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/serial-tool.qrc"/>
 </resources>
 <connections/>
</ui>
